### PR TITLE
BDP-1014: ${test.arguments} -> test.arguments

### PR DESCRIPTION
**Pull request by courtsVII**

### CODE REVIEW

**Review by massdosage — APPROVED** [review]

**Review by massdosage — CHANGES_REQUESTED** [review]
It now has merge conflicts so I'll leave you to resolve those before re-approving.

**Review by massdosage — APPROVED** [review]
Hopefully we can get this released today, otherwise that date in the changelog is going to need to be updated ;)

**Review by ddcprg — APPROVED** [review]

File: CHANGELOG.md
Comment (ddcprg) [inline CHANGELOG.md]: you've got 2 approvals, just don't forget to update this date before the merge and release

**Pull request merged**
